paddle/operators/detail/grpc_server.cc

```diff
@@ -132,6 +132,8 @@ void AsyncGRPCServer::RunSyncUpdate() {
 
   cq_send_ = builder.AddCompletionQueue();
   cq_get_ = builder.AddCompletionQueue();
+  cq_batch_barrier_ = builder.AddCompletionQueue();
```

Comment:
`cq_batch_barrier_` seems never used.

Reply:
Done, sorry I forgot to delete the older code.